api/samples/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ else (WIN32)
 endif (WIN32)
 
 if (DEBUG)
-  set(OPT_CFLAGS "-DDEBUG")
+  set(OPT_CFLAGS "${OPT_CFLAGS} -DDEBUG")
```

Comment:
We certainly want to fix the regression from e884280 which removed the optimization flags.

As for setting -DDEBUG I am ok either way, but if we do set it, we need to set it for standalone building outside of DR's containing cmake code, as mentioned.